### DISCUSSION TITLE
Support for newer versions of JDK

### DISCUSSION THREAD

**Discussion by machnst1** [OxalisCommunity/oxalis#676] [Q&A]
What is the timeline for Oxalis to add support for the newer version of java that are supported.
Notably, when is version 17 supported.

**Comment by teedjay** [comment]
Did you mean Java 17 or 21 ?

Java 18 was End Of Life a couple of years ago and only 11 / 17 / 21 have long term support.  Maybe some details will be shared in the Oxalis Community Technical Dialog Meeting this Wednesday, seems "Status on Road map" is part of the agenda.

**Reply by machnst1** [reply]
Ah, sorry. It was a typo, the correct version I meant was 17.

**Comment by dladlk** [comment]
The only obstacle to run Oxalis with Java 13+ was forbidden SHA1, which was used by SMP specification before 1.3.0. Since 01.06.2024 it is mandatory: 
![image](https://github.com/OxalisCommunity/oxalis/assets/41484081/6eae4640-a0ee-45ab-b946-e462e280bcbd)
We run Oxalis 6 instance on Java 21 since November 2023 (it required security.policy modification to allow SHA1) - but now even this is not required, so I would say Oxalis 6.5.0 ALREADY supports Java 21 as SHA1 is not used by SMPs anymore.
